### PR TITLE
Comment update improvements

### PR DESCRIPTION
**Pull request by levlaz**
This fixes https://github.com/kanboard/kanboard/issues/2934

* Added date_modification to comments table
* Sort by date_modification instead of date_created 
* Show created and update dates in the UI 
* Added unit and integration tests (confirmed passing locally)

### CODE REVIEW

File: tests/units.postgres.xml [hunk 1]
```diff
@@ -10,4 +10,4 @@
         <const name="DB_PASSWORD" value="" />
         <const name="DB_NAME" value="kanboard_unit_test" />
     </php>
-</phpunit>
\ No newline at end of file
+</phpunit>
```
Comment (levlaz) [inline tests/units.postgres.xml]: Bah! Sorry I can't seem to get this file back to the original state. :/

File: app/Template/comment/show.php [hunk 2]
```diff
@@ -7,7 +7,9 @@
             <strong class="comment-username"><?= $this->text->e($comment['name'] ?: $comment['username']) ?></strong>
         <?php endif ?>
 
-        <small class="comment-date"><?= $this->dt->datetime($comment['date_creation']) ?></small>
+        <small class="comment-date">Created At: <?= $this->dt->datetime($comment['date_creation']) ?></small>
+        <small class="comment-date">Updated At: <?= $this->dt->datetime($comment['date_modification']) ?></small>
```
Comment (fguillot) [inline app/Template/comment/show.php]: The strings "Created At:" and "Updated At:" should be translatable.

File: app/Schema/Postgres.php [hunk 3]
```diff
@@ -6,7 +6,12 @@
 use Kanboard\Core\Security\Token;
 use Kanboard\Core\Security\Role;
 
-const VERSION = 97;
+const VERSION = 98;
+
+function version_98(PDO $pdo)
+{
+    $pdo->exec('ALTER TABLE "comments" ADD COLUMN date_modification INT');
```
Comment (fguillot) [inline app/Schema/Postgres.php]: You should use a BIGINT for Postgres as well, in this way we are ready for https://en.wikipedia.org/wiki/Year_2038_problem

File: app/Template/comment/show.php (hunk 2 above)
Comment (levlaz) [inline app/Template/comment/show.php]: Thanks! Fixed.

File: app/Schema/Postgres.php (hunk 3 above)
Comment (levlaz) [inline app/Schema/Postgres.php]: Updated the migration file to do this.

File: app/Schema/Postgres.php [hunk 4]
```diff
@@ -6,7 +6,15 @@
...
 use Kanboard\Core\Security\Role;
 
-const VERSION = 97;
+const VERSION = 98;
+
+function version_98(PDO $pdo)
+{
+    $pdo->exec('ALTER TABLE "comments" ADD COLUMN date_modification BIGINT');
+    $pdo->exec('UPDATE "comments"
+        SET date_modificaiton = date_creation
```
Comment (fguillot) [inline app/Schema/Postgres.php]: Be careful, there is a typo here that prevent the migration to work :)

**Pull request merged**